src/main/java/org/fundacionjala/virtualassistant/models/RequestEntity.java

```diff
@@ -12,6 +12,8 @@ public class RequestEntity {
 
   @Column(name = "id_user")
   private long idUser;
+  @Column(name = "id_context")
+  private long idContext;
   @Column(name = "text")
   private String text;
   @Column(name = "date")
```

Comment:
here use a ZonedDateTime instead Date on the type

Reply:
int the long idAudioMongo use a wrapper Long instead of a primitive value of long